web/flask/static/js/task_status_page.js

```diff
@@ -52,6 +53,10 @@ function send_task_status_req() {
                 elem.children().eq(0).children().eq(1).text(task_id);
                 elem.children().eq(1).children().eq(1).text(task_status);
                 elem.children().eq(2).children().eq(1).text(task_msg);
+                elem.children().eq(3).children().eq(1).text("");
+                if(task_link!=""){
+                    elem.children().eq(3).children().eq(1).text('http://'+$("#hds_server").val()+task_link);
```

Comment:
Suggest to use `a` tag to make it clickable
```suggestion
                    elem.children().eq(3).children().eq(1).text('http://'+$("#hds_server").val()+task_link);
elem.children().eq(3).children().eq(1).attr('href', 'http://'+$("#hds_server").val()+task_link);
```